Makefile

```diff
@@ -67,13 +71,18 @@ algod-start:
 algod-stop:
 	docker compose stop algod
 
-integration-run:
+test-integ-async:
 	pytest -n $(NUM_PROCS) --durations=10 -sv tests/integration -m "not serial"
-	pytest --durations=10 -sv tests/integration -m serial
 
-test-integration: integration-run
+# Run tests w/ @pytest.mark.serial under ~/tests/integration each in its own proc:
+test-integ-sync:
+	find tests/integration -name '*_test.py' | sort | xargs -t -I {} pytest --suppress-no-test-exit-code --dist=no --durations=10 {} -m serial
```

Comment:
Very subtle balancing act for running each syncronous test in its own proc and still failing in the right way when a real test case fails